runner/src/main.rs

```diff
@@ -162,84 +162,32 @@ fn build_wasm_module(name: &str, manifest_path: &str) -> Step {
     }
 }
 
-fn is_mac_os() -> bool {
-    cfg!(macos)
-}
-
-fn bazel_build_flags() -> Vec<String> {
```

Comment:
Does the removal of this mean that C++ client applications are (already) built without the benefit of a remote cache?

Reply:
Yes, I may add it back separately.